modRlp/src/org/aion/rlp/RLP.java

```diff
@@ -396,47 +193,60 @@ public static void fullTraverse(byte[] msgData, int level, int startPos, int end
                     int length = calcLength(lengthOfLength, msgData, pos);
 
                     // now we can parse an item for data[1]..data[length]
-                    System.out.println("-- level: [" + level + "] Found big list length: " + length);
+                    System.out.println(
+                            "-- level: [" + level + "] Found big list length: " + length);
```

Comment:
Does this be printed to the terminal during kernel running? If yes, can we add new logEnum for the RLP module?

Reply:
I removed this functionality in #572 because it wasn't used.